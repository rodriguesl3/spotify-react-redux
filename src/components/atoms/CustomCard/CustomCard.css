.custom-card {
  background-color: black;
  margin: 20px 20px 20px 40px;
  padding: 20px;
  border-radius: 25px;
  border: gray 2px solid;
}

.card-img-top {
  width: 111%;
  border-top-left-radius: calc(.25rem - 1px);
  border-top-right-radius: calc(.25rem - 1px);
  position: absolute;
  left: -34px;
  border-radius: 25px;
  height: 160px;
}
/*# sourceMappingURL=CustomCard.css.map */