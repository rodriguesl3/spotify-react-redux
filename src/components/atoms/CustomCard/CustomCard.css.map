{
    "version": 3,
    "mappings": "AAGA,AAAA,YAAY,CAAC;EACT,gBAAgB,ECMR,KAAK;EDLb,MAAM,EAAE,mBAAmB;EAC3B,OAAO,EAAE,IAAI;EACb,aAAa,EAAE,IAAI;EACnB,MAAM,ECDM,IAAI,CDCO,GAAG,CAAC,KAAK;CACnC;;AAED,AAAA,aAAa,CAAC;EACV,KAAK,EAAE,IAAI;EACX,sBAAsB,EAAE,kBAAkB;EAC1C,uBAAuB,EAAE,kBAAkB;EAC3C,QAAQ,EAAE,QAAQ;EAClB,IAAI,EAAE,KAAK;EACX,aAAa,EAAE,IAAI;EACnB,MAAM,EAAE,KAAK;CAChB",
    "sources": [
        "CustomCard.scss",
        "../../../base.scss"
    ],
    "names": [],
    "file": "CustomCard.css"
}