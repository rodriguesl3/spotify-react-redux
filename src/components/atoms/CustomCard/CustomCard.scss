@import "../../../base.scss";


.custom-card {
    background-color: $base-color;
    margin: 20px 20px 20px 40px;
    padding: 20px;
    border-radius: 25px;
}

.card-img-top {
    width: 111%;
    border-top-left-radius: calc(.25rem - 1px);
    border-top-right-radius: calc(.25rem - 1px);
    position: absolute;
    left: -34px;
    border-radius: 25px;
    height: 160px;
}