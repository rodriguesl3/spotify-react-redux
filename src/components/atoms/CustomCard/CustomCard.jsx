import React from 'react';
import './CustomCard.scss';

export const CustomCard = (props) => {
  const { image, content } = props;

  return (
    <div className="col-10 col-sm-4 col-lg-3 custom-card">
      <div className="row">
        <div className="col-5">
          <img src={image} className="card-img-top" alt="card-name" />
        </div>
        <div className="col-7">
          {content}
        </div>
      </div>
    </div>
  )
};

