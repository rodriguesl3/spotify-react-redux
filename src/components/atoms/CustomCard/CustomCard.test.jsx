/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import { CustomCard } from './CustomCard';

test('CustomCard snaphot test', () => {
  const component = renderer.create(<CustomCard image="https://cdn-images-1.medium.com/max/900/1*G2QwxPF2TvWXzRUnA4axoA.png" content="teste" />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
