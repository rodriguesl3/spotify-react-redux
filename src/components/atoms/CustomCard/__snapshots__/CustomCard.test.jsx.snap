// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomCard snaphot test 1`] = `
<div
  className="col-10 col-sm-4 col-lg-3 custom-card"
>
  <div
    className="row"
  >
    <div
      className="col-5"
    >
      <img
        alt="card-name"
        className="card-img-top"
        src="https://cdn-images-1.medium.com/max/900/1*G2QwxPF2TvWXzRUnA4axoA.png"
      />
    </div>
    <div
      className="col-7"
    >
      teste
    </div>
  </div>
</div>
`;
